core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -441,7 +441,8 @@ private[kafka] class Processor(val id: Int,
     false,
     true,
     ChannelBuilders.serverChannelBuilder(listenerName, securityProtocol, config, credentialProvider.credentialCache),
-    memoryPool)
+    memoryPool,
+    new LogContext())
```

Comment:
In `SocketServer`, we have:

```scala
this.logIdent = "[Socket Server on Broker " + config.brokerId + "], "
```

We could set the `logContext` as a field in `SocketServer`, use it to set `logIdent` and then pass it here. We can probably use `[SocketServer brokerId=${config.brokerId}] ` as the prefix.